tooling/minimize/tests/pass/locks.stdout

```diff
@@ -0,0 +1,2 @@
+0
+1
```

Comment:
Isn't the order non-deterministic here?

Reply:
No. The lock is held by the main thread before the other thread is spawned. It will only release it once it has printed.

Reply:
Whenever there's something non-obvious like this going on, a comment in the code is required. :)